modules/services/mpd.nix

```diff
@@ -198,6 +198,8 @@ in {
 
       Install = { WantedBy = [ "sockets.target" ]; };
     };
+
+    home.packages = [ cfg.package ];
```

Comment:
May be possible to do a more targeted install using `getMan`. I.e. something like

```
home.packages = [ (getMan cfg.package) ];
```